priv/assets/javascripts/benchee.js

```diff
@@ -86,12 +86,14 @@ window.drawComparisonBoxPlot = function(runTimes, sortOrder, inputHeadline) {
   drawGraph(boxNode, boxPlotData(runTimes, sortOrder), layout);
 };
 
-window.drawRawRunTimeCharts = function(runTimes, inputHeadline) {
+window.drawRawRunTimeCharts = function(runTimes, statistics, inputHeadline) {
   var runTimeNode = document.getElementById("raw-run-times");
   var jobName = runTimeNode.getAttribute("data-job-name");
+  var minY = (statistics.minimum) - (0.1 * statistics.minimum);
```

Comment:
Wouldn't this logic be the same as `statistics.minimum * 0.9`? To me that seems like a clearer way to express what we're going for here.

Reply:
Yes that is a much less confusing representation of the arithmetic and makes it much more readable